Makefile

```diff
@@ -6,14 +6,17 @@ boilerplate-update:
 
 SHELL := /usr/bin/env bash
 
-FORWARDER_VERSION=$(shell cat .splunk-version)
-FORWARDER_HASH=$(shell cat .splunk-version-hash)
+FORWARDER_VERSION ?= 8.0.5
+FORWARDER_HASH ?= a1a6394cc5ae
+SFI_HASH_7 ?= fa50892
```

Comment:
Wouldn't hurt to have a comment describing what this is, as it's not super intuitive.
```suggestion
# Short hash of the commit in openshift/splunk-forwarder-images that produced the splunk-forwarder and splunk-heavyforwarder images the operator will deploy.
SFI_HASH_7 ?= fa50892
```

...but looking closer, I think it would actually be appropriate to get rid of these as separate variables. They never *need* to be used individually, and disaster would ensue if any of them weren't in perfect sync with each other.

In other words:
- Scrap `FORWARDER_VERSION`, `FORWARDER_HASH`, and `SFI_HASH_7`
- Make `FORWARDER_IMAGE_TAG` "hardcoded"
- Make `update-image-vars.sh` just replace the `FORWARDER_IMAGE_TAG` with the computed `image_tag`; and invoke `make image-digests` with that one value instead of three.